distribution/helm/vector-aggregator/values.yaml

```
@@ -175,6 +175,16 @@ logSchema:
   sourceTypeKey: "source_type"
   timestampKey: "timestamp"
 
+# The "built-in" api , for healthchecks  and graphql requests.
+# Will be disabled by default.
+vectorApi:
+  # Disable to omit the vector api from being added.
+  enabled: false
+  # The address to listen at.
+  address: "127.0.0.1:8686"
```

```suggestion
  address: "0.0.0.0:8686"
```
Makes more sense to use this out of the box, rather than loopback interface.

---

agree.